src/extension.ts

```diff
@@ -90,6 +91,9 @@ class SarifExtension implements Api {
         this.logReader = new LogReader();
         context.subscriptions.push(this.logReader);
 
+        // Register our file system provider.
+        context.subscriptions.push(new ArtifactContentFileSystemProvider());
```

Comment:
What does this have to do with keyboard navigation?

Reply:
Ah.. nothing. This was removed accidentally when I merged the last PR to master.

Result => artifact content objects (like binary content) fail to render (oops).

---
In reply to: [413139862](https://github.com/microsoft/sarif-vscode-extension/pull/254#discussion_r413139862) [](ancestors = 413139862)